src/main/java/com/minecraftabnormals/abnormals_core/core/util/DataUtil.java

```diff
@@ -314,4 +330,93 @@ public static void registerConfigPredicate(IConfigPredicateSerializer<?> seriali
 			throw new IllegalStateException("Duplicate config predicate serializer: " + key);
 		ConfigValueCondition.Serializer.CONFIG_PREDICATE_SERIALIZERS.put(key, serializer);
 	}
-}
+
+	/**
+	 * When an instance of this class is registered using {@link DataUtil#registerAlternativeDispenseBehavior(AlternativeDispenseBehavior)},
+	 * an {@link IDispenseItemBehavior} will be registered that will perform a new {@link IDispenseItemBehavior} if
+	 * a condition is met and the behavior that was already in the registry if not. See constructor for details.
+	 *
+	 * <p>This works even if multiple mods
+	 * add new behavior to the same item, though it is possible that the conditions might overlap, which is what
+	 * {@code modComparator} is intended to solve.</p>
```

Comment:
it is -> it's
Double use of uncertainty
```suggestion
	 * <p>This works even if multiple mods
	 * add new behavior to the same item, though the conditions may overlap, which is what
	 * {@code modComparator} is intended to solve.</p>
```